src/main/java/gregtech/loaders/recipe/chemistry/NuclearRecipes.java

```diff
@@ -16,22 +63,277 @@ public static void init() {
+                .input(dust, Magnesium, 2)
+                .output(dust, Zirconium, 1)
+                .output(dust, MagnesiumChloride, 6)
+                .buildAndRegister();
+
+        BLAST_RECIPES.recipeBuilder().duration(200).EUt(VA[EV]).blastFurnaceTemp(1150)
+                .input(dust, HafniumTetrachloride, 5)
+                .input(dust, Magnesium, 2)
+                .output(dust, Hafnium, 1)
+                .output(dust, MagnesiumChloride, 6)
+                .buildAndRegister();
+
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[EV])
+                .input(dust, Zirconium, 16)
+                .fluidInputs(Tin.getFluid(32))
+                .fluidInputs(Chrome.getFluid(2))
+                .output(dust, Zircaloy, 16)
+                .buildAndRegister();
+
+        // Inconel 718
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[EV])
+                .input(dust, Nickel, 5)
+                .input(dust, Chrome, 2)
+                .input(dust, Iron, 2)
+                .fluidInputs(Niobium.getFluid(72))
+                .fluidInputs(Molybdenum.getFluid(48))
+                .output(dust, Inconel, 10)
+                .buildAndRegister();
+
+        // LEU-235 Dioxide
+        CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .fluidInputs(HighEnrichedUraniumHexafluoride.getFluid(1000))
+                .fluidInputs(Water.getFluid(2000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .output(dust, HighEnrichedUraniumDioxide, 1)
+                .fluidOutputs(HydrofluoricAcid.getFluid(6000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .fluidInputs(LowEnrichedUraniumHexafluoride.getFluid(1000))
+                .fluidInputs(Water.getFluid(2000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .output(dust, LowEnrichedUraniumDioxide, 1)
+                .fluidOutputs(HydrofluoricAcid.getFluid(6000))
+                .buildAndRegister();
+
+        CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
                 .fluidInputs(DepletedUraniumHexafluoride.getFluid(1000))
-                .output(dust, Uranium238)
-                .fluidOutputs(Fluorine.getFluid(6000))
+                .fluidInputs(Water.getFluid(2000))
+                .fluidInputs(Hydrogen.getFluid(2000))
+                .output(dust, DepletedUraniumDioxide, 3)
+                .fluidOutputs(HydrofluoricAcid.getFluid(6000))
+                .buildAndRegister();
+
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .input(dust, HighEnrichedUraniumDioxide, 1)
+                .input(dust, DepletedUraniumDioxide, 5)
+                .output(dust, HEU235, 6)
+                .buildAndRegister();
+
+        MIXER_RECIPES.recipeBuilder().duration(200).EUt(VA[LV])
+                .input(dust, HighEnrichedUraniumDioxide, 1)
```

Comment:
According to the materials, which Zalgo set up, no. Low-enriched uranium dioxide, I would assume, is hardly more enriched than natural uranium, so it would not be suitable.